### PR TITLE
doc: fix re-arrangement of docs

### LINKED ISSUE #3981: how to upgrade from 1.0 to 1.1 ?
*Description*:
[I Install v1.0 with YAML](https://gateway.envoyproxy.io/v1.0/install/install-yaml/)

Currently, I see that version 1.1 is released. I want to upgrade to 1.1, but I can't find any upgrade documents to refer to?

### PR DESCRIPTION
**Pull request by octonawish-akcodes**
Re-arranged the docs to find out how to upgrade from `v1.0` to `v1.1`
Fixes #3981

### CODE REVIEW

File: site/content/en/v1.1/install/install-yaml.md [hunk 1]
```diff
@@ -37,3 +37,31 @@ Refer to the [Developer Guide](../../contributions/develop) to learn more.
...
+Delete `BackendTLSPolicy` CRD (and resources): 
+
+```shell
+kubectl delete crd backendtlspolicies.gateway.networking.k8s.io
+```
+
+Update Gateway-API and Envoy Gateway CRDs:
+
+```shell
+helm pull oci://docker.io/envoyproxy/gateway-helm --version v1.1.0 --untar
```
Comment (arkodg) [inline site/content/en/v1.1/install/install-yaml.md:54]: lets template the version `< yaml-version >`

File: site/content/en/v1.1/install/install-yaml.md [hunk 2]
```diff
@@ -37,3 +37,31 @@ Refer to the [Developer Guide](../../contributions/develop) to learn more.
...
+```
+
+Update your `BackendTLSPolicy` and `GRPCRoute` resources according to Gateway-API [v1.1 Upgrade Notes](https://gateway-api.sigs.k8s.io/guides/#v11-upgrade-notes)
+
+Update your Envoy Gateway xPolicy resources: remove the namespace section from targetRef. 
+
+Install Envoy Gateway v1.1.0:
+
+```shell
+helm upgrade eg oci://docker.io/envoyproxy/gateway-helm --version v1.1.0 -n envoy-gateway-system
```
Comment (arkodg) [inline site/content/en/v1.1/install/install-yaml.md:66]: can we template the versions with `< yaml-version >` ?

File: site/content/en/v1.1/install/install-yaml.md [hunk 3]
```diff
@@ -37,3 +37,31 @@ Refer to the [Developer Guide](../../contributions/develop) to learn more.
 2. Next Steps
 
    Envoy Gateway should now be successfully installed and running, but in order to experience more abilities of Envoy Gateway, you can refer to [Tasks](/latest/tasks).
+
+## v1.1 Upgrade Notes
```
Comment (arkodg) [inline site/content/en/v1.1/install/install-yaml.md:41]:
```suggestion
## Upgrading from v1.0 to v1.1
```

**Comment by arkodg** [comment]
@zirain @zhaohuabing does this doc need to live in multiple places or does @octonawish-akcodes need to run additional make commands to make sure the tool generates those ?

**Comment by zirain** [comment]
> @zirain @zhaohuabing does this doc need to live in multiple places or does @octonawish-akcodes need to run additional make commands to make sure the tool generates those ?

you need to update `latest` and `v1.1` and `run make -k gen-check`

**Pull request closed without merging**